src/Joystick/Joystick.cc

```diff
@@ -278,8 +278,12 @@ void Joystick::run(void)
             float   pitch = _adjustRange(_rgAxisValues[axis], _rgCalibration[axis]);
             
                     axis = _rgFunctionAxis[yawFunction];
+
             float   yaw = _adjustRange(_rgAxisValues[axis], _rgCalibration[axis]);
-            
+                    _rgCalibration[axis].min = -32768;
```

Comment:
Could you please only do this for axis 0, assuming its the only one which gets destroyed?